app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -214,6 +214,7 @@ public void onFinish() {
   private boolean wasHideToolbar = true;
   private boolean isHideToolbar = true;
   private boolean isSpeaking = false;
+  private int currentTabPosition;
```

Comment:
I think you can use `currentWebViewIndex` instead of defining a new variable.

Reply:
Done